src/main/java/com/sfuentes/Theater.java

```diff
@@ -0,0 +1,44 @@
+package com.sfuentes;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Theater {
+
+  ArrayList<Integer> ages;
+  final double price = 20d;
+
+  public Theater(ArrayList<Integer> ages) {
+    this.ages = ages;
+  }
+
+  public Map<String, Double> getCategoryByDiscount() {
+    double discount1 = 0;
+    double discount2 = 0;
+    double discount3 = 0;
+    double discount4 = 0;
+    double discount5 = 0;
+    for (int age : ages) {
+      if (age >= 5 && age <= 14)
+        discount1 += price * .35;
+      else if (age >= 15 && age <= 19)
+        discount2 += price * .25;
+      else if (age >= 20 && age <= 45)
+        discount3 += price * .10;
+      else if (age >= 46 && age <= 65)
+        discount4 += price * .25;
+      else if (age >= 66)
+        discount5 += price * .35;
+    }
+    Map<String, Double> total = new HashMap<>();
+    total.put("One category", discount1);
```

Comment:
You can name the categories accordingly:
"5-14"
"15-19"
etc
so it would be more clear

Reply:
fix

Reply:
fix